assets/js/dist/fade.js

```diff
@@ -1,40 +1,38 @@
 "use strict";
 
-$(document).ready(function () {
-  var mobileOffset = 270; // Start with only work section showing
+var mobileOffset = 270; // Start with nav-work button highlighted and only work section showing
 
-  $('#nav-work').removeClass('default-btn').addClass('current-btn');
-  $('#work').addClass('display');
-  var elements = document.getElementsByClassName('nav-btn');
-  Array.from(elements).forEach(function (el) {
-    el.addEventListener('click', function (e) {
-      e.preventDefault();
-      $('li').removeClass('current-btn');
-      $("#".concat(e.target.id)).addClass('current-btn');
-      $('#fade-wrapper').fadeOut(500, function () {
-        $('section').removeClass('display');
-        var sectionId = e.target.id.slice(4);
-        $("#".concat(sectionId)).addClass('display');
+document.getElementById('nav-work').classList.add('current-btn');
+document.getElementById('work').classList.add('display');
+var navElements = document.getElementsByClassName('nav-btn');
+Array.from(navElements).forEach(function (navElement) {
+  navElement.addEventListener('click', function (e) {
+    e.preventDefault();
+    Array.from(document.getElementsByTagName('li')).forEach(function (el) {
```

Comment:
This selector seems too generic. It's operating on all list items, not just list items in the header.